src/WorkItemFiling/AzureDevOpsClientWrapper.cs

```diff
@@ -13,21 +13,20 @@
 using Microsoft.VisualStudio.Services.WebApi.Patch.Json;
 using Newtonsoft.Json;
 
-namespace Microsoft.WorkItemFiling
+namespace Microsoft.WorkItems
 {
     /// <summary>
     /// Represents an Azure DevOps project in which work items can be filed.
     /// </summary>
-    public class AzureDevOpsClientWrapper : FilingClient
+    public class AzureDevOpsClientWrapper: FilingClient
     {
         private WorkItemTrackingHttpClient _witClient;
-        
-        public override async Task Connect(string personalAccessToken)
```

Comment:
>personalAccessToken [](start = 50, length = 19)

Good name change. I think I saw other places in the code where "personalAccessToken" remains.

Reply:
Well, both ADO and GitHub actually use the 'personal access token' term. So I'm not sure about this change (because we won't be accepting arbitrary security tokens here, rather, this data is specifically for PATs.

---
In reply to: [381027919](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381027919) [](ancestors = 381027919)

Reply:
For AzureDevOps, we are currently authenticating users with a basic token, if we are going to do service to service auth, that would be a bearer token. I believe GitHub uses a similar model, I think we should be consistent and crystal clear to the users what type of token we are taking in to reduce less confusion. So keeping it as personalAccessToken would be helpful to the end users I believe.

---
In reply to: [382847231](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r382847231) [](ancestors = 382847231,381027919)